src/main/resources/languages.txt

```diff
@@ -1,62 +0,0 @@
-af_ZA=Afrikaans (ZA)
```

Comment:
If we are going to add in-game language selection someday this might be useful though.

Well, useless for now. I agree.